easybuild/easyconfigs/b/beagle-lib/beagle-lib-2.1.2-goolf-1.7.20.eb

```diff
@@ -2,6 +2,7 @@ easyblock = 'ConfigureMake'
 
 name = 'beagle-lib'
 version = '2.1.2'
+versionsuffix = '-Java-%(javaver)s'
```

Comment:
@pescobar did you mean to change an existing easyconfig?

It's not going to work anyway, missing versionsuffix in the filename...